### ISSUE TITLE
Missing flutter.js in generated branch

### ISSUE THREAD

**Issue by ghost** [kevmoo/peanut.dart#129]
Running the build I often find that the gh-pages branch is missing the flutter.js file that is needed to serve the flutter page. This does not appear to happen deterministically.

Steps used to push the branch:

```
$ flutter pub global run peanut
$ git push origin --set-upstream gh-pages
```

* OS: Windows 11
* Flutter version: 3.0.1

**Comment by kevmoo** [comment]
This is likely due to a bug in FLUTTER, not peanut. I hit the same issue. Let me go dig up the bug.

**Comment by kevmoo** [comment]
Fix included in peanut v4.2.0 – just published!
